source/angular-date-picker.js

```diff
@@ -22,18 +19,28 @@ angular.module( "vokal.datePicker", [] )
 
                     ngModelController.$setValidity( "date", !isNaN( dateData.getTime() ) );
 
-                    return dateData;
+                    return attrs.pickerType === "string" ?
+                        $filter( "date" )( dateData, attrs.datePicker || "M/d/yyyy" ) : dateData;
```

Comment:
Keep the default format `"M/d/yyyy"` in a variable since it's used in a couple locations.